ovs_gtp/roles/ovs_gtp/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+# ---
+# # Copyright (c) 2016-present, Facebook, Inc.
+# # All rights reserved.
+# #
+# # This source code is licensed under the BSD-style license found in the
+# # LICENSE file in the root directory of this source tree. An additional grant
+# # of patent rights can be found in the PATENTS file in the same directory.
+# #
+- name: Removing "{{WORK_DIR}}".
+  file:
+    path: "{{WORK_DIR}}"
+    state: absent
+
+- name: Creating "{{WORK_DIR}}".
+  file:
+    path: "{{WORK_DIR}}"
+    state: directory
+    mode: 0755
+
+- name: Install build dependencies.
+  become: yes
+  apt:
+    name: "{{ packages }}"
+  vars:
+    packages:
+      - graphviz
+      - debhelper
+      - dh-autoreconf
+      - python-all
+      - python-twisted-conch
+      - module-assistant
+      - openssl
+      - pkg-config
+      - libssl-dev
+      - build-essential
+      - libcap-ng-dev
+      - git
+      - ruby-dev
+
+- name: Ensure correct kernel headers are installed.
+  shell: "apt -y install linux-headers-$(uname -r)"
+  become: yes
+
+- name: Install fpm.
+  become: yes
+  gem:
+    name: fpm
+    state: present
+
+- name: Cloning the ovs "{{ovs_version}}".
+  git:
+    repo: https://github.com/kkroo/ovs.git
+    dest: "{{WORK_DIR}}/ovs"
+    version: "{{ovs_version}}"
+
+- name: Cloning the flow-based-gtp-linux-v4.9.
+  git:
+    repo: https://github.com/ouyangjn/flow-based-gtp-linux-v4.9.git
```

Comment:
Can you do something similar for this one too which is to keep this in the magma github repo? the goal at the end of this patch is that we don't depend on some other github repo